src/transform2d.rs

```diff
@@ -522,11 +522,21 @@ impl <T, Src, Dst> Default for Transform2D<T, Src, Dst>
 }
 
 impl<T: ApproxEq<T>, Src, Dst> Transform2D<T, Src, Dst> {
+    /// Returns true is this transform is approximately equal to the other one, using
+    /// T's default epsilon value.
     pub fn approx_eq(&self, other: &Self) -> bool {
         self.m11.approx_eq(&other.m11) && self.m12.approx_eq(&other.m12) &&
         self.m21.approx_eq(&other.m21) && self.m22.approx_eq(&other.m22) &&
         self.m31.approx_eq(&other.m31) && self.m32.approx_eq(&other.m32)
     }
+
+    /// Returns true is this transform is approximately equal to the other one, using
+    /// a provided epsilon value.
+    pub fn approx_eq_eps(&self, other: &Self, eps: T) -> bool {
```

Comment:
the matrix API receives epsilon by reference. Is there a strong reason to break consistency at this level?

Reply:
Sorry that was me hesitating and forgetting to clean it up. I'd rather we consistently pass it by value but I don't care enough to change it.